src/CommunityVoices/Public/js/quote-collection.js

```diff
@@ -153,3 +153,12 @@ $('.approve-checkbox').on('click', function (c) {
 		}
 	});
 });
+
+$('#fileUploadButton').on('click', function (c)  {
+	alert("Guide To Uploading: Upload 2 files, one with quote information, one with source data information. \nBoth should be in csv format. Ensure that quote sheet has word 'quote' and that source sheet has word 'source' contained within.");
```

Comment:
"Contained within" makes me think within the context of the file.  I'd rather say "filename includes" just for more clarity.

Reply:
Adjusted - let me know if you prefer otherwise